telegram/_bot.py

```diff
@@ -2037,16 +2051,40 @@ async def send_media_group(
                 :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
             api_kwargs (:obj:`dict`, optional): Arbitrary keyword arguments to be passed to the
                 Telegram API.
+            group_caption (:obj:`str`, optional): Text of the caption
+                that will be shown for the whole media group.
+                Defaults to :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
```

Comment:
```suggestion
                Defaults to :obj:`None`.
```

Or even skip it completely, as that's already implied by the `, optional)` :)

Reply:
fixing in commit e4b7971164c4c08aab6920401bae0bd74397dcf7 (I also add some text, not sure it's good). 

Don't know what's best for the docs - to put `Defaults to` or to skip :)